### DISCUSSION TITLE
MPD database format

### DISCUSSION THREAD

**Discussion by jedediahg** [MusicPlayerDaemon/MPD#1680] [Q&A]
Is there any documentation on the MPD database format? I'm interested in directly reading the DB in order to perform more advanced searches to automate some tagging tools and can't find any details on the tag_cache file format.

**Comment by MaxKellermann** [comment]
It is intentional that there is no documentation. The format is an implementation detail that changes often. Use the MPD protocol to query it.

**Reply by jedediahg** [reply]
Thanks very much for the quick reply! I want to do things like `SELECT WHERE %artist% != %albumartist%` and I don't think that can be done with MPD protocol via MPC, or can it?

**Reply by sp1ff** [reply]
Check out mpdpopm
